packages/psss/src/routes/PageRoutes.js

```diff
@@ -14,30 +14,35 @@ import { PrivateRoute } from './PrivateRoute';
 import { UnauthorisedView } from '../views/UnauthorisedView';
 import { ProfileView } from '../views/ProfileView';
 import { NotFoundView } from '../views/NotFoundView';
-import { canUserViewMultipleCountries, getCurrentUser, canUserViewCountry } from '../store';
+import { getEntitiesAllowed, canUserViewMultipleCountries, canUserViewCountry } from '../store';
 
-export const PageRoutesComponent = React.memo(({ user }) => (
+export const PageRoutesComponent = React.memo(({ allowedEntities }) => (
   <Switch>
-    <PrivateRoute exact path="/" authCheck={() => canUserViewMultipleCountries(user)}>
+    <PrivateRoute
+      exact
+      path="/"
+      authCheck={() => canUserViewMultipleCountries(allowedEntities)}
+      redirectTo={`/weekly-reports/${allowedEntities[0]}`}
```

Comment:
redirect to the appropriate weekly reports page if it is a user with only access to a single country